packages/browser/src/browser/index.ts

```diff
@@ -105,6 +105,13 @@ function hasLegacyDestinations(settings: LegacySettings): boolean {
   )
 }
 
+function hasTsubMiddleware(settings: LegacySettings): boolean {
+  return (
+    getProcessEnv().NODE_ENV !== 'test' &&
+    (settings.middlewareSettings?.routingRules?.length ?? 0) > 0
```

Comment:
Minor: This change will does the same thing - and should shave bytes (I think `??` is transpiled to be a lot more code than expected)
```suggestion
    settings.middlewareSettings?.routingRules?.length
```

Reply:
Ah - type signature wants a boolean so you'd need to wrap the whole thing in a `Boolean` or `!!`, sorry - not sure it's worth the shaved bytes.